### ISSUE TITLE
Multiple inheritance types render incomplete in method response

### ISSUE THREAD

**Issue by tiguchi** [mulesoft/api-console#363]
# Problem
Given the following minimal example `api.raml` file:
```
#%RAML 1.0
title: Test API
types:
  BaseA:
    properties:
      a:
        type: string
  BaseB:
    properties:
      b:
        type: string
  Complete:
    type: [ BaseA, BaseB ]
    properties:
      c:
        type: string

/complete:
  get:
    responses:
      200:
        body:
          application/json:
            type: Complete
```
Custom object types that inherit from multiple base types appear incomplete in any resource method response section. Above example only shows properties inherited from `BaseA`and the properties directly defined in the type `Complete`:

![image](https://cloud.githubusercontent.com/assets/6569525/20361379/37472b58-abec-11e6-9bca-fd85bf3be501.png)

The order of the `type` array in `Complete` matters. When `BaseB` comes first, then only B's properties appear in the output, on top of the properties defined in `Complete`.

The output in the documentation's `Types` section is complete though:

![image](https://cloud.githubusercontent.com/assets/6569525/20361476/8d2e7a9e-abec-11e6-9af1-3e4aa980e882.png)

# System & Version Info
API Console Version: 3.0.10
Browser: Google Chrome Version 54.0.2840.98 (64-bit)
OS: Mac OS X Sierra 10.12.1

**Comment by sichvoge** [comment]
Can you try with the latest version? It works for me again

![image](https://cloud.githubusercontent.com/assets/498114/20606704/a0fec0ea-b26a-11e6-9732-12d1b8e26d06.png)

**Comment by tiguchi** [comment]
Yes, I can confirm that the output appears complete. However, resource type popups (clicking one of the blue links) still contain incomplete information (description and id properties are missing):

![image](https://cloud.githubusercontent.com/assets/6569525/20607020/f76165c8-b229-11e6-9292-82c24d35e2cd.png)

**Comment by sichvoge** [comment]
Seems you have a different RAML in your image. Using your RAML you posted here I can see all necessary information. Can you share exactly what you have or a small example on that with step by step explanation to reproduce it?

**Comment by tiguchi** [comment]
Sorry, that was a "real life" example screen shot.  I modified the original example from above to reproduce the problem:

```
#%RAML 1.0
title: Test API
types:
  BaseA:
    properties:
      a:
        type: string
  BaseB:
    properties:
      b:
        type: string
  ReferencedComposite:
    type: [ BaseA, BaseB ]
    properties:
      r:
        type: string

  Complete:
    type: [ BaseA, BaseB ]
    properties:
      c:
        type: ReferencedComposite

/complete:
  get:
    responses:
      200:
        body:
          application/json:
            type: Complete
```

"ReferencedComposite" is missing `b`:

![image](https://cloud.githubusercontent.com/assets/6569525/20607775/f2f994aa-b22f-11e6-9701-d1824349ef8e.png)

**Comment by nmarinel01** [comment]
Reproducible using 3.0.11. Thank you !

**Comment by nmarinel01** [comment]
Fixed at 3.0.16 release
